src/core/client/admin/routes/Configure/HorizontalRule.tsx

```diff
@@ -1,9 +1,15 @@
+import cn from "classnames";
 import React, { FunctionComponent } from "react";
 
 import styles from "./HorizontalRule.css";
 
-const HorizontalRule: FunctionComponent = ({ children }) => (
-  <hr className={styles.root} />
+interface Props {
+  className?: string;
+  children?: any;
+}
+
+const HorizontalRule: FunctionComponent<Props> = ({ children, className }) => (
+  <hr className={cn(styles.root, className)} />
```

Comment:
Do not use this `<HorizontalRule />` component inside the `Moderate` route as this is from another route. Replace usage of `<HorizontalRule />` with an `<hr />` or a new `<HorizontalRule />` for the `Moderate` route.